synapse-get-annotations-tool.cwl

```diff
@@ -0,0 +1,53 @@
+
+#!/usr/bin/env cwl-runner
+#
+# Authors: Andrew Lamb
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: 
+  - synapse
+  - get-annotations
+
+hints:
+  DockerRequirement:
+    dockerPull: sagebionetworks/synapsepythonclient:v1.9.2
+
+inputs:
+
+  - id: synapse_config
+    type: File
+
+  - id: synapseid
+    type: string
+    inputBinding:
+      prefix: --id
+
+  - id: output_file_name
```

Comment:
I know the command line tool provides this, but use `stdout` instead as it's a more standard interface.